canonical-web-and-design/ubuntu.com
# Add GetUbuntu redirects

## Done

- Add GetUbuntu redirects as we were getting 404s

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/GetUbuntu and https:/0.0.0.0:8001/getubuntu
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are redirected to /download
